pages/docs/column-types/mysql.mdx

```diff
@@ -538,6 +557,17 @@ CREATE TABLE `table` (
 ```
 </Section>
 
+### Customizing column data type
+
+Every column builder has a `.$type()` method, which allows you to customize the data type of the column. This is useful, for example, with branded types.
+
+```ts
+const users = mysqlTable('users', {
+  id: serial('id').$type<UserId>().primaryKey(),
```

Comment:
I suppose we need to declare branded type above and then show example usage below